xcode/Subconscious/Shared/Components/Common/Profile/FollowUserFormSheet.swift

```diff
@@ -11,18 +11,20 @@ import ObservableStore
 import CodeScanner
 
 struct FollowNewUserFormSheetView: View {
-    var state: FollowNewUserFormSheetModel
+    var store: ViewStore<FollowNewUserFormSheetModel>
+    var did: Did?
```

Comment:
We should get this value into `FollowNewUserFormSheetModel` somehow instead of having to thread in an additional value.